### PR TITLE
Add a hook for ``pingouin`` to collect data files.

### PR DESCRIPTION
**Pull request by JonMrowczynski**

### CODE REVIEW

**Comment by bwoodsend** [comment]
Are users really likely to need those files? The csv files look like test/example datasets (although they are pretty tiny so they don't waste much space if they're not needed).

**Comment by JonMrowczynski** [comment]
Hello Brénainn,

Thank you very much for your response.hi

The reason for adding those files was due to at least pingouin's need for them. When I attempted to create either a onedir or a onefile executable using PyInstaller for a Python script using pingouin, it would fail when the executable was run. When my script imports pingouin, a FileNotFoundError would be thrown because pingouin would try to load dataset.csv...which wouldn't get packaged without the hook.

Adding this hook was the only way I was able to get my script to run as an executable when implementing pingouin. Kind of like what was recommended here: https://groups.google.com/g/pyinstaller/c/OYhJdeZ9010

Let me know if that answers your question or if I could offer any additional information. Thanks! :)
________________________________

**Comment by bwoodsend** [comment]
I've tried it out and it looks like only the `pingouin/datasets/datasets.csv` file is needed but it's much easier just to collect everything like you already are so I'll accept this for now. Can you give us [a test](https://github.com/bwoodsend/pyinstaller-hooks-contrib/tree/readme#write-tests) and [add pingouin to the requirements file](https://github.com/bwoodsend/pyinstaller-hooks-contrib/tree/readme#pin-the-test-requirement)?

**Comment by JonMrowczynski** [comment]
Hello again Brénainn, That’s what I was thinking as well. Thank you very much for taking a look at that. I just added a test and pingouin to the requirements file. I also pushed those changes to my master branch. Let me know if you need anything else from me. Thanks again. From: Brénainn WoodsendSent: Wednesday, August 18, 2021 12:24To: pyinstaller/pyinstaller-hooks-contribCc: Jon Mrowczynski; AuthorSubject: Re: [pyinstaller/pyinstaller-hooks-contrib] Add a hook for ``pingouin`` to collect data files. (#292) I've tried it out and it looks like only the pingouin/datasets/datasets.csv file is needed but it's much easier just to collect everything like you already are so I'll accept this for now. Can you give us a test and add pingouin to the requirements file?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Review by bwoodsend — APPROVED** [review]

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py [hunk 1]
```diff
@@ -678,3 +678,9 @@ def test_rtree(pyi_builder):
     pyi_builder.test_source("""
         import rtree
         """)
+
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/tests/test_libraries.py:681]: Ugh damm, the linter wants an extra blank line here.
```suggestion


```

**Review by bwoodsend — CHANGES_REQUESTED** [review]

**Comment by bwoodsend** [comment]
Thanks @JonMrowczynski!

**Comment by JonMrowczynski** [comment]
No problem, thank you! 😊 From: Brénainn WoodsendSent: Thursday, August 19, 2021 9:52To: pyinstaller/pyinstaller-hooks-contribCc: Jon Mrowczynski; MentionSubject: Re: [pyinstaller/pyinstaller-hooks-contrib] Add a hook for ``pingouin`` to collect data files. (#292) Thanks @JonMrowczynski!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**